src/modules/operations/bundle/buildWrapTx.ts

```diff
@@ -0,0 +1,13 @@
+import { Weth } from 'legacy/abis/types'
+
+export type BuildWrapTxParams = {
+  wrappedNativeContract: Weth
+  weiAmount: string
+}
+
+/**
+ * Builds a wrap tx for the given order, without sending it
+ */
+export async function buildWrapTx({ wrappedNativeContract, weiAmount }: BuildWrapTxParams) {
```

Comment:
Do we really need the function? I see it's used only once

Reply:
Right now, maybe not.
Still, I want to isolate these type of functions for when we get to the fully featured complex batch building.
Same for the other fns in this module